app/src/main/java/org/simple/clinic/drugs/selection/entry/CustomPrescriptionEntrySheet.kt

```diff
@@ -61,7 +61,11 @@ class CustomPrescriptionEntrySheet : BottomSheetActivity() {
   }
 
   override fun onBackgroundClick() {
-    if (drugNameEditText.text.isBlank() && drugDosageEditText.text.isBlank()) {
+    val drugNameEmpty = drugNameEditText.text.isEmpty()
+    val dosageEmpty = drugDosageEditText.text.isEmpty()
+        || drugDosageEditText.text.toString() == getString(R.string.customprescription_dosage_placeholder)
```

Comment:
Do we need to ignore whitespaces when checking for the placeholder string?

Reply:
Done